includes/class-fs-storage.php

```diff
@@ -26,6 +26,10 @@ class FS_Storage {
          * @var FS_Storage[]
          */
         private static $_instances = array();
+        /**
+         * @var bool|int
+         */
+        public $install_timestamp;

```

Comment:
@DanieleAlessandra would it also work if you just add a `@property bool|int $install_timestamp` in the PHP doc block, instead of this change?

The reason I am asking is, let's not make changes to the existing system, unless needed.

Reply:
Maybe it will work, I try...